styles/style.scss

```diff
@@ -0,0 +1,465 @@
+@import './import';
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body {
+    background-color: #000000;
+    color: white;
+}
+h1 {
+    text-align: center;
+    margin: 3.75rem 0 2.5rem;
+    span {
+        background-image: linear-gradient(to right, #20D4FE , #B722FF);
+        background-clip: text;
+        -webkit-text-fill-color: transparent;
+    }
+    @media screen and (max-width: 1440px) {
+        font-size: 3.5rem;
+        margin: 2.5rem 0 1.9375rem;
+    }
+    @media screen and (max-width: 1280px) {
+        font-size: 3.125rem;
+        margin: 3.125rem 0 1.875rem;
+    }
+    @media screen and (max-width: 1024px) {
+        font-size: 2.5rem;
+        margin: 2.5rem 0 1.875rem;
+    }
+    @media screen and (max-width: 991px) {
+        font-size: 2.1875rem;
+    }
+    @media screen and (max-width: 768px) {
+        font-size: 1.875rem;
+    }
+    @media screen and (max-width:575px) {
+        font-size: 2rem;
+        margin: 3.5rem 0 2rem;
+    }
+    @media screen and (max-width: 480px) {
+        font-size: 1.75rem;
+        margin: 2.5rem 0 1.5rem;
+    }
+    @media screen and (max-width: 375px) {
+        max-width: 12.1875rem;
+        font-size: 2rem;
+        padding: 1.5rem 0 1rem;
+        margin: auto;
+    }
+}
+.table-wrapper {
+    margin: 0 23.125rem;
+        @media screen and (max-width: 1440px) {
+            margin: 0 11.25rem;
+        }
+        @media screen and (max-width: 1280px) {
+            margin: 0 10.4375rem;
+        }
+        @media screen and (max-width: 1024px) {
+            margin: 0 4.375rem;
+        }
+        @media screen and (max-width: 575px) {
+            height: auto;
+            overflow-x: auto;
+            white-space: nowrap;
+            margin: 0 2.1875rem;
+        }
+        @media screen and (max-width: 375px) {
+            margin: 0 1rem;
+        }
+    .table {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        @media screen and (max-width: 575px) {
+            display: inline-block;     
+        }
+        .table-cell {
+            &:first-child {
```

Comment:
refactor code 
remove nth child and add classes for widths like 
.w-14 {
width: "14%;
}

Reply:
done